ext/liquid_c/parser.c

```diff
@@ -128,12 +128,9 @@ static void parse_and_compile_variable_lookup(parser_t *p, vm_assembler_t *code)
             parser_must_consume(p, TOKEN_CLOSE_SQUARE);
             vm_assembler_add_lookup_key(code);
         } else if (p->cur.type == TOKEN_DOT) {
-            int has_space_affix = parser_consume_any(p).flags & TOKEN_SPACE_AFFIX;
```

Comment:
I think this makes TOKEN_SPACE_AFFIX, TOKEN_SPACE_PREFIX and TOKEN_SPACE_SUFFIX unused, so I think we can remove them now.